XamarinCommunityToolkit/Core/UriMediaSource.shared.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Core
+{
+	public sealed class UriMediaSource : MediaSource
+	{
+		public static readonly BindableProperty UriProperty = BindableProperty.Create(nameof(Uri), typeof(Uri), typeof(UriMediaSource), default(Uri),
+			propertyChanged: (bindable, oldvalue, newvalue) => ((UriMediaSource)bindable).OnSourceChanged(), validateValue: (bindable, value) => value == null || ((Uri)value).IsAbsoluteUri);
```

Comment:
don't you mind to extract all delegates into separate static methods?

Reply:
done